beacon-chain/rpc/beacon/attestations.go

```diff
@@ -250,6 +250,18 @@ func (bs *Server) StreamIndexedAttestations(
 				}
 				bs.ReceivedAttestationsBuffer <- data.Attestation
 			}
+			if event.Type == operation.AggregatedAttReceived {
+				data, ok := event.Data.(*operation.AggregatedAttReceivedData)
+				if !ok {
+					// Got bad data over the stream.
+					continue
+				}
+				if data.Attestation == nil || data.Attestation.Aggregate == nil {
+					// One nil attestation shouldn't stop the stream.
+					continue
+				}
+				bs.CollectedAttestationsBuffer <- []*ethpb.Attestation{data.Attestation.Aggregate}
```

Comment:
Can you change this to `ReceivedAttestationsBuffer` please, if its `Collected` then the slasher will get spammed with every attestation received.

Reply:
can we revert it back?